policies/trustee_roles_and_responsibilities.md

```diff
@@ -0,0 +1,127 @@
+* Regular participation in discussions on Slack
+  * This is one of the main channels of Trustee discussion and people should be prepared to accept notifications of urgent discussions at all times and check in to the main discussion thread every few days
+  * Slack is also used to make urgent decisions that cannot wait until the next board meeting
+* Be an ambassador for Farset Labs and it’s values (internally and externally)
+* Management of communications channels (email, facebook, twitter)
+* Co-ordinating the [Annual General Meeting](../runbooks/rituals/annual_general_meeting.md)
+* Contributing to the Annual Trustees Report for submission with our annual accounts
+
+### Specific tasks and responsibilities of Secretary, Treasurer & Chair
+
+_Note - these are functional roles only and do not confer any additional authority_
+
+* [Chair](https://www.diycommitteeguide.org/code/principle/what-role-chairperson)
+  * Representing Farset Labs in an official capacity when necessary
+  * Chairing the Annual General Meeting and Trustees Meetings
+  * Monitoring and Validating that decisions taken at meetings are implemented
+* [Secretary](https://www.diycommitteeguide.org/code/principle/secretary)
+  * Setting agenda and recording minutes from board and members meetings
+  * Ensuring we meet our legal obligations with the Charities Commission for Northern Ireland and Companies House
+  * Draft agendas working with the other trustees and chair
+    – Take minutes in meetings or organise cover where necessary
+    – Receive correspondence and organise responses in a timely fashion
+    – Keep the registers of Trustees up to date with Companies House and the Charities Commission for Northern Ireland
+* [Treasurer](https://www.diycommitteeguide.org/code/principle/treasurer)
+  * Coordinating and monitoring budgets and managing finances
+  * Regularly reconciling our accounting balances
+  * Liaising with our accountant
+  * Ensuring we meet our financial obligations with Companies House and HMRC and that all submission deadlines are met
+    – Reminding members to consider the financial implications of decisions they are taking and not to take decisions unless they have sufficient information in front of them
+
+It is also important to note that as a trustee, [you will not be able to be paid by Farset Labs for any services or goods that you wish to provide the charity](https://www.charitycommissionni.org.uk/manage-your-charity/making-payments-to-trustees/). This restriction also extends to any members of your immediate family and is a legal obligation. [Expenses may be reimbursed under certain circumstances.](financial_management.md)
+
+
+### Purpose of Board Meetings
+
+Quarterly board meetings shall cover
+
+* Strategic planning
+* Financial planning & management
+* Programme and Subcommittee reports and oversight
+* Issues escalated from any subcommittees or working groups
```

Comment:
Subcommittees or working groups such as...? I get that this is probably a catch-all in case subgroups ever arise but it could be worth clarifying or scrapping